docs/research/user-experience/xmr-btc-user-interface-validation.md

```diff
@@ -0,0 +1,117 @@
+# XMR<>BTC Atomic Swap - User Interface Prototype Validation
+
+This document:
+
+1. Collects the validation criteria.
+2. Lists the created user interface prototypes, and link to Figma.
+3. Maps the protoypes to the validation criteria.
+
+This document will be updated with new information during the course of the project.
+
+## Questions
+
+The questions are split between `M`aker (liquidity provider) and `T`aker (normal user), because the objectives are somewhat different.
+
+|  **Topic** | **High Level Questions** | **More specific question** | **User is happy to...** | **Actor** |
+| --- | --- | --- | --- | --- |
+|  Node & Wallet Management | How do users monitor the Bitcoin and Monero blockchain? | Is a user fine with trusting a third party to monitor transactions? | use a service like blockchain.com to retrieve blocks for validation | TM |
+|   |  |  | run his own Bitcoin node, third party service for Monero | TM |
+|   |  |  | run his own Monero node, third party service for Bitcoin | TM |
+|   |  |  | run both his own Bitcoin and Monero node | TM |
+|   | How do users brodcast transactions to Bitcoin and Monero? | Is a user fine with trusting a third party to broadcast transactions? | use a wallet that connects to third party nodes | TM |
+|   |  |  | send signed transactions through third part nodes | TM |
+|   |  |  | run his own Bitcoin node, third party service for Monero | TM |
+|   |  |  | run his own Monero node, third party service for Bitcoin | TM |
+|   |  |  | run both his own Bitcoin and Monero node | TM |
```

Comment:
``````suggestion
   |  Node & Wallet Management | How do users monitor the Bitcoin and Monero blockchains? | Is a user fine with trusting a third party to monitor transactions? | use a third party service to retrieve transactions | TM |
   |   |  |  | run his own blockchain node | TM |
   |   | How do users brodcast transactions to Bitcoin and Monero? | Is a user fine with trusting a third party to broadcast transactions? | use a wallet that connects to third party nodes | TM |
   |   |  |  | send signed transactions through third part nodes | TM |
   |   |  |  | run his own full blockchain node | TM |
   |   |  |  | run a SPV node (Bitcoin) | TM |
   ```

IMHO, This adds quite a bit of text with little added value. I was not going to comment as I thought I was nitpicking but then I saw the same pattern for the wallet and think it is a bit too much.

I think it's clear enough in the way I propose. Feel free to disagree 🙃

Reply:
I was not sure about this one myself - I opted for distinguishing the sides, because users might actually consider that.
Reasoning: A monero user already has Monero set up but might not care about setting up and running a Bitcoin full node. Since I am just not sure if that applies or not I thought I keep it separate for now, so it can be validated that users are willing to run both nodes.

Adding the SPV node is an interesting option! It shows, once again, that there are quite a lot of options, and making them mutually exclusive is hard 😬

Reply:
I'll adapt it to what you propose! We just have to keep in mind to record the user answers for both sides. I would reckon that a person that cares about running a Monero full node also cares about running a bitcoin full node, but when it e.g. comes to Ethereum that is not true. Most people would trust Infura for that.